src/cli/common.go

```diff
@@ -125,11 +126,31 @@ func ValidateSchemas(projectState *state.State, logger *zap.SugaredLogger) error
 	return nil
 }
 
-func Rename(projectState *state.State, logger *zap.SugaredLogger) error {
-	if warn, err := local.Rename(projectState.RenamePlan(), logger); err != nil {
+func Rename(projectState *state.State, logger *zap.SugaredLogger, logEmpty, dryRun bool) error {
+	// Get plan
+	rename := plan.Rename(projectState)
```

Comment:
Rename opracia uz poziva tiez podobny plan ako `push/pull`, je to jednotne a da sa jednoducho spravit `dry-run`.